b2dEditor.py

```diff
@@ -0,0 +1,347 @@
+import pygame
+from tkinter import Tk
+from tkinter import filedialog
+import json
+import sys
+
+resolution = (800, 600)
+
+return_json = ""
+
+background = None
+
+def set_background():
+    global background
+
+    image_file_name = filedialog.askopenfilename(filetypes=[("PNG files", "*.png")], initialdir="~")
+    if image_file_name == "" or image_file_name == ():
+        return
+    background = pygame.image.load(image_file_name)
+    background.convert_alpha()
+    background.fill((255, 255, 255, 125), None, pygame.BLEND_RGBA_MULT)
+    background = pygame.transform.scale(background, (edit_area.rect[2] - 50, edit_area.rect[3] - 50))
+
+if "--help" in sys.argv:
+    print("\nUsage: python3 b2dEditor.py [--help] [--resolution <selection>]\n\n"
+          "--help:       Prints this help text\n"
+          "--resolution: Sets application resolution to selected option: \n"
+          "     1: 800x600\n"
+          "     2: 1200x900\n"
+          "     3: 1800x1350\n")
+    sys.exit(0)
+
+if "--resolution" in sys.argv:
+    resolution_arg = sys.argv[sys.argv.index("--resolution") + 1]
+    if resolution_arg == "1": resolution = (800, 600)
+    elif resolution_arg == "2": resolution = (1200, 900)
+    elif resolution_arg == "3": resolution = (1800, 1350)
+
+shapes = []
+current_shape = 0
+
+shapes.append([])
+
+buttons = []
+
+quit = False
+selected = None
+Tk().withdraw()
+
+def dump_node_json():
+    global return_json, quit
+    if __name__ == "__main__":
```

Comment:
Would possibly like to see a parameter instead of `__name__` checks. It's more robust and versatile